perses/app/setup_relative_calculation.py

```diff
@@ -381,11 +383,22 @@ def run_setup(setup_options, serialize_systems=True, build_samplers=True):
     else:
         measure_shadow_work = False
         _logger.info(f"\tno measure_shadow_work specified: defaulting to False.")
-
-    pressure = setup_options['pressure'] * unit.atmosphere
-    temperature = setup_options['temperature'] * unit.kelvin
-    solvent_padding_angstroms = setup_options['solvent_padding'] * unit.angstrom
-    ionic_strength = setup_options['ionic_strength'] * unit.molar
+    if isinstance(setup_options['pressure'],float):
```

Comment:
for lines like this, would it make more sense to do this unit checking stuff in fah_generator.py after the yaml is read in? otherwise i will need to add lines like this into the PointMutationExecutor too

Reply:
It needs to be be fixed in a lot of places, which is why I suggested https://github.com/choderalab/perses/issues/730

just fixing it in the fah_pipeline, doesn't fix it for repex/sams/any simulation not on folding at home